ENVParser/Utils/JsonExporter.cs

```diff
@@ -0,0 +1,121 @@
+﻿using ENVParser.Fields;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace ENVParser.Utils
+{
+    internal class JsonExporter
+    {
+
+        private readonly HashSet<string> _validFieldsForRGBValues;
+        private readonly JsonOutput output;
```

Comment:
Looks like you don't actually use this (`output`)...? Should it be deleted, then?

Reply:
Correct, I noted this in one of my later updates, and have removed it accordingly.